CppWinRTProjectionSample/SimpleMathProjection/SimpleMathProjection.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <Platforms>x64</Platforms>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GenerateTestProjection>true</GenerateTestProjection>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!--<NoWarn>8305;0618</NoWarn>-->
+    <GeneratedFilesDir>.\Generated Files\</GeneratedFilesDir>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="0.1.0-prerelease.200922.1" />
```

Comment:
I'd avoid this - all samples and unit tests should import the cswinrt build files (props, targets) directly, and copy winrt.runtime.dll directly

Reply:
@Scottj1s - I'm not sure I follow. this is a sample showing what an end developer would do to build a projection for their cppwinrt component - this is not a test for the repo itself. In this case we want to show how to get the latest cswinrt - through the public nuget package.

Reply:
we can finish this PR and retrofit it (along with other samples) to be buildable against the local nuget, overriding use of the public nuget so users can simply copy

Reply:
referencing the latest public nuget - [Microsoft.Windows.CsWinRT 0.1.0-prerelease.200925.12](https://www.nuget.org/packages/Microsoft.Windows.CsWinRT/0.1.0-prerelease.200925.12)